README.md

```diff
@@ -26,12 +40,18 @@ differences:
 
 ## Source Configuration
 
-* `repository`: *Required.* The name of the repository, e.g. `alpine`. If using ecr
-    you only need the repository name, not the full URI e.g. `alpine` not
-    `012345678910.dkr.ecr.us-east-1.amazonaws.com/alpine`
+* `repository`: *Required.* The name of the repository, e.g. `alpine` or
+  `concourse/concourse`.
+
+  *Note: If using ecr you only need the repository name, not the full URI e.g.
+  `alpine` not `012345678910.dkr.ecr.us-east-1.amazonaws.com/alpine`*
+
+* `tag`: *Optional.* Instead of monitoring semver tags, monitor a single tag
+  for changes (based on digest).
 
-* `tag`: *Optional. Default `latest`.* The name of the tag to monitor and
-  publish to.
+* `variant`: *Optional.* Detect only tags matching this variant suffix, and
```

Comment:
It might be obvious, but *maybe* it's worth pointing out that this doesn't do anything if you have `tag` set, and that you should instead add the variant to the `tag`.

e.g. you'd specify:

```
tag: stable-alpine
```

rather than

```
tag: stable
variant: alpine
```

Reply:
It technically still does something - it won't be used for `check`, but if someone does a `put` with a `version` set, the configured variant will be appended to the version:

```yaml
resources:
- name: my-image
  type: registry-image
  source:
    repository: vito/foobar
    tag: banana
    variant: alpine

jobs:
- name: pusha-t
  plan:
  - put: my-image
    params: {version: 1.2.3} # pushes 'banana' and '1.2.3-alpine'
```

It's a little weird, but it's a thing.

Pushed a README change, lemme know what you think!

Reply:
Ah thanks, makes sense